andela/bad_ass-ah-backend
# #165525643 fix email verification and login after verifying email account

**What does this PR do?**

This PR fixes bugs for sending email verification and prevent a user to log in without verifying his/her email account

**Description of Task to be completed?**

- User signup to the application
- Before doing anything including to log in, he/ she have to verify his/her email first.

**How should this be manually tested?**

After cloning the repo, install all the packages by running ```npm install```
- Signup as a new user
- You immediately get an email in your account for verifying your email account.
- Log in using the credentials you used while signing up.

**Any background context you want to provide?**

Before the application was allowing the user to log in without verifying his/her email account but now the user should verify it before login.

**What are the relevant pivotal tracker stories?**

[#165525643](https://www.pivotaltracker.com/story/show/165525643)
